internal/api/courses/assignments/submissions/analysis/individual.go

```diff
@@ -16,10 +16,12 @@ type IndividualRequest struct {
 }
 
 type IndividualResponse struct {
-	Complete bool                                 `json:"complete"`
-	Options  analysis.AnalysisOptions             `json:"options"`
-	Summary  *model.IndividualAnalysisSummary     `json:"summary"`
-	Results  map[string]*model.IndividualAnalysis `json:"results"`
+	Success    bool                                 `json:"success"`
```

Comment:
`model.AnalysisSummary` already has a `FailureCount` (which is properly maintained).
I think we should add an `ErrorCount` to summarize work errors.

I think we should probably also drop `Success` here.
Feel free to disagree, but I think its just a bit too awkward in too many cases, e.g.,:
 - 1,000,000 jobs completed successfully, but one had a work error.
 - There were some failures.
 - There were 100% failures.
 - Nothing was run.

`Success` is not "wrong" in these cases, just awkward.